docs/exports_yaml_files.md

```diff
@@ -121,6 +122,36 @@ Example in `app/views/exports/duplex_seq_pcr_xp_concentrations_for_custom_poolin
 ancestor_purpose: LDS AL Lib Dil
 ```
 
+#### filename
+
+A method of specifiying what data is included in the filename of the export.
+
+```yaml
+filename:
+  # The name of the export file
+  name: 'example-name'
+  # Includes the page number at the end of the filename
+  include_page: true
+  # Includes the labware's human barcode in the file
+  labware_barcode:
+    # Includes the name of the labware in the file
+    append: true
+    # Add the name to the end of the file
+    prepend: false
+  # Includes the parent labware's human barcode
+  parent_labware_barcode:
+    append: true
+    prepend: true
```

Comment:
Are there any defaults here or, if you include `labware_barcode`, do you have to specify `true`/`false` for `append` AND `prepend` ?

Reply:
Neither are required or have defaults. Ill update the comments to reflect that.